docs/architecture/grpc-for-wcf-developers/toc.yml

```diff
@@ -21,8 +21,6 @@
       href: interface-definition-language.md
     - name: Network protocols
       href: network-protocols.md
-    - name: WCF bindings and transports
```

Comment:
why is this article being removed from the TOC? The file is still in this PR.

Reply:
I'm going to guess this this a mistake. We don't want to omit this file

Reply:
Ok reverting the change then